### DISCUSSION TITLE
Your site no longer works

### DISCUSSION THREAD

**Discussion by cometstyles** [lovasoa/dezoomify#782] [General]
Your dezoomify site no longer works as in can no longer dezoomify images fro ma site which i used to do easily for the last 7 years. its giving the error of not being able to find a zoomable image at the address i gave but i use the browser extension and it gives the correct link, only your site no longer can dezoomify it. I have used that site atleast 40 times on your site to get zoomable images but it no longer works now..

**Comment by lovasoa** [comment]
Hello and welcome to the issues section !

  The dezoomify website itself still works the same as it always has. If I had to guess, I'd say it's the website from which you make downloads that made technical changes.
   You can open an issue with all the technical details. Just saying "it does not work" without even a link to the website you are talking about doesn't really help :)
